### ISSUE TITLE
L.PM.Utils circleToPolygon

### ISSUE THREAD

**Issue by gmarshall56** [geoman-io/leaflet-geoman#968]
The api states:
circleToPolygon(circle, sides = 60)

When the Circle draw button is clicked and the user draws a circle on the map, can the program code use the properties from e.layer, such as _mRadius or _latlng when calling this utility program, or does the code first have to instantiate a L.Circle variable using these same parameters and then use the L.Circle variable when calling the circleToPolygon utility?

Basically I am asking how to use this utility.  What will function as the "circle" parameter?  Do you have to instantiate a Circle first or can you use a property from e.layer as the "circle" parameter?

Thank you

**Comment by Falke-Design** [comment]
In the function a `L.Circle` needs to be passed. You can create a new circle `var circle = L.circle(coords, {radius: RADIUS})` or simply pass the `e.layer` as circle:

```
map.on('pm:create', (e)=>{
    L.PM.Utils.circleToPolygon(e.layer);
})
```

**Comment by gmarshall56** [comment]
Thank you.  I assume I can use the same e.layer in the L.PM.Utils.circleToPolygon(e.layer) if the user were to click the CircleMarker draw button?

Thank you
